src/Tokenizers/Tokenizer.php

```diff
@@ -80,8 +80,7 @@ abstract class Tokenizer
      * @param string                         $eolChar The EOL char used in the content.
      *
      * @return void
-     * @throws \PHP_CodeSniffer\Exceptions\TokenizerException If the file appears to be minified.
-     */
+	 */
```

Comment:
Something looks off about the alignment here....
```suggestion
     */
```

Reply:
Actually, on second thought, I'd leave this one (for now) as two out of three of the Tokenizer sub-classes throw that Exception from the overload `__construct()` method.